shared/libraries/websocket_streaming/src/streaming_client.cpp

```diff
@@ -243,7 +243,7 @@ void StreamingClient::parseConnectionString(const std::string& url)
     std::smatch match;
 
     // parsing connection string to four groups: prefix, host, port, path
-    auto regexIpv6Hostname = std::regex(R"(^(.*://)?(?:\[([a-fA-F0-9:]+)\])(?::(\d+))?(/.*)?$)");
+    auto regexIpv6Hostname = std::regex(R"(^(.*://)?(?:\[([a-fA-F0-9:]+(?:\%\s+)?)\])(?::(\d+))?(/.*)?$)");
```

Comment:
auto regexIpv6Hostname = std::regex(R"(^(.*://)?(?:\[([a-fA-F0-9:]+(?:\%\w+)?)\])(?::(\d+))?(/.*)?$)");

Reply:
Missed that one.. Good catch.